package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@launchpadlab/lp-components",
-  "version": "5.1.1",
+  "version": "5.3.0",
```

Comment:
Let me know if you think this should be a major version update instead. While we did change an existing interface, the change only added to the interface so the change is completely backwards compatible.

Reply:
Backwards compatible -> minor change is good!